receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_linux.go

```diff
@@ -30,3 +37,33 @@ var allTCPStates = []string{
 	"SYN_RECV",
 	"TIME_WAIT",
 }
+
+func scrapeAndAppendConntrackMetrics(s *scraper, metrics pdata.MetricSlice) error {
+	now := pdata.TimestampFromTime(time.Now())
+
+	conntrack, err := s.conntrack()
+	if err != nil {
+		return err
```

Comment:
Add context to the error
```suggestion
		return fmt.Errorf("failed to scrape conntrack metrics: %w", err)
```